ecs-init/config/common.go

```diff
@@ -147,3 +147,12 @@ func DockerUnixSocket() (string, bool) {
 func CgroupMountpoint() string {
 	return cgroupMountpoint
 }
+
+// HostCertsDirPath() returns the CA store path on the host (specifically on
+// Amazon Linux platform)
```

Comment:
nit: This isn't specific to Amazon Linux, the configured value may be, I think this comment could simply trim off the parenthetical and remain perfectly clear.